web_server.py

```
@@ -0,0 +1,319 @@
+
+    try:
+        event_id = int(request.path_params["event_id"])
+    except ValueError:
+        return JSONResponse({"error": "Invalid event_id"}, status_code=400)
+
+    try:
+        removed = db.delete_event(event_id)
+        if removed:
+            return JSONResponse({"ok": True})
+        return JSONResponse({"error": "Event not found"}, status_code=404)
+    except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)